lib/font-fit.js

```diff
@@ -1,5 +1,25 @@
 ;(function(define){'use strict';define(function(require,exports,module){
 
+  /* This map holds values specific to a heading.
+   * The key is the heading node itself. Because it's a WeakMap, it should be
+   * freed by the GC when the node is removed from the DOM.
+   */
+  var privMap = new WeakMap();
+
+  function getPriv(instance) {
```

Comment:
This seems like quite a complicated way to cache `start` and `end` attributes? Can't we just store them on the heading instance?

```js
heading.dataset.start = start;
heading.dataset.end = end;
```